src/components/pages/ImageClassification/ImageAnnotationModal/ImageAnnotationPopup/ImageAnnotationPopup.js

```diff
@@ -62,30 +62,29 @@ const ImageAnnotationPopup = ({
 
   return (
     <>
-      <form>
```

Comment:
Just removed the form tag here because all the collect pages are already wrapped in a form, so its semantically incorrect....